tests/config/test_config_comparator.py

```diff
@@ -16,7 +16,7 @@
 from tests.config.utils.unique_section_for_tests import UniqueSectionForTest
 
 
-def test_config_comparator():
+def test_config_comparator_return():
```

Comment:
The test function `test_config_comparator_return` is long and difficult to understand. The name does not help, and it s difficult to find a better name since we test many things on it.

What about splitting the test into multiple tests like:
- test_comparator_without_diff
- test_comparator_with_updated_global_config
- test_comparator_with_new_section
- test_comparator_with_updated_section
- test_comparator_with_deleted_section
- test_comparator_with_new_unique_section
- test_comparator_with_updated_unique_section
- test_comparator_with_deleted_unique_section